src/test/test-utils.ts

```diff
@@ -155,3 +156,5 @@ export function resolvedUrl(
     strings: TemplateStringsArray, ...values: any[]): ResolvedUrl {
   return noOpTag(strings, ...values) as ResolvedUrl;
 }
+
+export const fixtureDir = path.join(__dirname, '../../src/test/static');
```

Comment:
not blocking, but why not make these a top level `fixtures/` or `static/` dir? Does it contain typescript files that need to be  compiled?

Reply:
We totally should. Now that we've got this referenced in one place it'd be a quick change. I'll do it separately